### ISSUE TITLE
Server Crashes when Generating

### ISSUE THREAD

**Issue by stacytritt** [Electroblob77/Wizardry#248]
``` Description: Exception ticking world

java.lang.IllegalArgumentException: Cannot calculate quantiles of an empty dataset
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.google.common.math.Quantiles$ScaleAndIndex.computeInPlace(Quantiles.java:277)
	at com.google.common.math.Quantiles$ScaleAndIndex.compute(Quantiles.java:266)
	at electroblob.wizardry.worldgen.WorldGenSurfaceStructure.findValidPosition(WorldGenSurfaceStructure.java:171)
	at electroblob.wizardry.worldgen.WorldGenSurfaceStructure.generate(WorldGenSurfaceStructure.java:215)
	at net.minecraftforge.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:167)
	at net.minecraft.world.chunk.Chunk.func_186034_a(Chunk.java:1020)
	at net.minecraft.world.chunk.Chunk.func_186030_a(Chunk.java:980)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:157)
	at net.minecraft.server.management.PlayerChunkMapEntry.func_187268_a(PlayerChunkMapEntry.java:126)
	at net.minecraft.server.management.PlayerChunkMap.func_72693_b(SourceFile:147)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:227)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:756)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:748)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.google.common.math.Quantiles$ScaleAndIndex.computeInPlace(Quantiles.java:277)
	at com.google.common.math.Quantiles$ScaleAndIndex.compute(Quantiles.java:266)
	at electroblob.wizardry.worldgen.WorldGenSurfaceStructure.findValidPosition(WorldGenSurfaceStructure.java:171)
	at electroblob.wizardry.worldgen.WorldGenSurfaceStructure.generate(WorldGenSurfaceStructure.java:215)
	at net.minecraftforge.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:167)
	at net.minecraft.world.chunk.Chunk.func_186034_a(Chunk.java:1020)
	at net.minecraft.world.chunk.Chunk.func_186030_a(Chunk.java:980)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:157)
	at net.minecraft.server.management.PlayerChunkMapEntry.func_187268_a(PlayerChunkMapEntry.java:126)
	at net.minecraft.server.management.PlayerChunkMap.func_72693_b(SourceFile:147)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:227)

-- Affected level --
Details:
	Level name: Biome Bundle
	All players: 1 total; [EntityPlayerMP['RigoBros'/464, l='Biome Bundle', x=-599.50, y=130.00, z=0.50]]
	Chunk stats: ServerChunkCache: 860 Drop: 0
	Level seed: 2643453385808176412
	Level generator: ID 06 - OTG, ver 0. Features enabled: true
	Level generator options: OpenTerrainGenerator
	Level spawn location: World: (0,80,0), Chunk: (at 0,5,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 66934 game time, 66934 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 89440 (now: false), thunder time: 9494 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:756)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:748)
```

Server crashes trying to generate new wizardry structures. Structures are disabled in config.

**Comment by Electroblob77** [comment]
As discussed on discord, you can avoid this crash by enabling fast worldgen. Since all the structures are disabled that won't make any difference anyway. I will get this fixed for the next update.

**Comment by Electroblob77** [comment]
I've looked into this further, and the only way this could have happened is if you deleted the structure files from inside the mod jar or changed the structure file list in the config (I suspect the latter is what's happened). This is not the correct way to disable structures, you should instead be using the structure dimension lists to disable them.

I will prevent the crash and add a warning message instead, and improve the config option descriptions.

**Comment by Electroblob77** [comment]
Fixed in 1e885d7
